### ISSUE TITLE
Xfce additions for 27

### ISSUE THREAD

**Issue by ghost** [kororaproject/kp#93]
~~The notification theme on Xfce is currently not set which results in a white colored notification pop up. This should be set to Default in the notification settings.~~

**Comment by jimkp** [comment]
As Menulibre is listed in Whiskermenu properties and the button is active we should include it. 
No settings changes would be needed but we should review the other settings in WhiskerMenu - Properties - Commands
(Maybe we should make this an Xfce additions issue)

**Comment by jimkp** [comment]
Replace lightdm-gtk with slick-greeter, as a result lightdm-gtk-greeter-settings is no longer needed.

(Also for Mate)

**Comment by DarkEra** [comment]
The first test iso i built today showed that the notification theme was as should, so we don't need to fix that one. We have to keep an eye on it if it stays that way during the dev cycle.

**Comment by DarkEra** [comment]
As i mentioned on IRC the icon or say launcher for dnfdragora went missing somehow only showing a questionmark icon. I also noticed that the trash applet has disappeared.
![missing trash applet and dnfdragora icon](https://user-images.githubusercontent.com/6264656/31560181-a54d8dba-b053-11e7-89c0-5560a40f4ee4.png)

**Comment by DarkEra** [comment]
"Synchronize drawing to the vertical blank" seems to be disabled under the Window Manager Tweaks --> Compositor tab. 

I know for sure we had it enabled by default. So something caused that our default korora xfce settings got overwritten which shouldn't happen.

**Comment by jimkp** [comment]
> As i mentioned on IRC the icon or say launcher for dnfdragora went missing somehow only showing a questionmark icon. I also noticed that the trash applet has disappeared.

This is due to the launcher properties being blank

**Comment by jimkp** [comment]
Review the font size for the panel clock. It looks smaller than other items on the panel.

**Comment by DarkEra** [comment]
Indeed, the current font size for the clock is set to 10 (as is on K26), 12 should be a better fit.

**Comment by jimkp** [comment]
with the recent update to lightdm-settings (for Slick greeter) numlockx is required for one option, to have numlock on so probably best to include it.

**Comment by DarkEra** [comment]
Done.

**Comment by DarkEra** [comment]
When you want to view a image on Xfce, Darktable opens as default. I guess this needs to be changed to Ristretto or Shotwell.

**Comment by jimkp** [comment]
Add a line to ~/.config/catfish/catfish.rc with 'use-headerbar=false"
this removes the csd header and makes catfish look like other Xfce applications
